.github/workflows/pytest-core-nompi.yml

```diff
@@ -38,17 +38,17 @@ jobs:
         ]
         set: [base, adjoint]
         include:
-        - name: pytest-ubuntu-py37-gcc5-omp
+        - name: pytest-ubuntu-py37-gcc11-omp
           python-version: '3.7'
-          os: ubuntu-18.04
-          arch: "gcc-5"
+          os: ubuntu-22.04
+          arch: "gcc-11"
```

Comment:
After discussions, keeping gcc-5 would be preferable as it is one of the critical ones, gcc-12 below is fine.